packages/dev/gui/src/2D/controls/control.ts

```diff
@@ -1821,9 +1821,16 @@ export class Control implements IAnimatable {
         }
     }
 
+    /**
+     * Ugly hack to skip first clipping because there is something wrong

```

Comment:
Sounds like there should be another way or we should find the root cause ???

Reply:
Thanks to @RaananW we found it! evaluateClippingState uses transformMatrix, but there was no guarantee that had been computed before the first evaluate call, so we added a _transform() before to calculate it.